weasyprint/css/validation.py

```diff
@@ -1267,6 +1269,23 @@ def bookmark_level(token):
         return 'none'
 
 
+@validator(prefixed=True)  # CSS3 GCPM, experimental
+def string_set(tokens):
```

Comment:
I have a hard time following this, sorry. Could you clarify what set of syntax you mean to support? All unsupported syntax should be rejected.